firefox_ui_harness/testcases/update.py

```diff
@@ -0,0 +1,133 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..testcase import FirefoxTestCase
+
+
+class UpdateTestCase(FirefoxTestCase):
+
+    def __init__(self, *args, **kwargs):
+        FirefoxTestCase.__init__(self, *args, **kwargs)
+
+        self.target_buildid = kwargs.pop('update_target_buildid')
+        self.target_version = kwargs.pop('update_target_version')
+
+        self.update_channel = kwargs.pop('update_channel')
+        self.update_mar_channels = set(kwargs.pop('update_mar_channels'))
```

Comment:
This is not true. UpdateOptions define a [default value](https://github.com/mozilla/firefox-ui-tests/pull/122/files#diff-d58666e1f5c3e602384c1489b217af51R16) which in this case is an empty array.